playgame.py

```diff
@@ -61,36 +66,61 @@ def prettyPrintInfo(info):
             print("-----")
             
         while not self.table.is_game_over():
-            bot = self.bots[self.currentPlayer]
-            info = self.table.info_for_player(self.currentPlayer)
+            bot = self.bots[self.current_player]
```

Comment:
This may not be the time or place to mention this, but having both `bot` and `player` implies differences, to my mind, that I'm not sure are real. Is there a reason for the distinction that I am not yet seeing?

Reply:
There isn't a reasonable difference here. `player` is the noun used for a game, while I used `bot` to distinguish as the ai strategies that are loaded to play the game. It should be `player`